### PR TITLE
Analyses Folder

### PR DESCRIPTION
**Pull request by DarthSethious1**
Now that we have an analysis in C++, we wanted to not have the problem of Arduino trying to make that an .ino file. This is also our 2nd analysis, so a separate 'analysis folder' was needed.

### CODE REVIEW

**Review by CClarke513 — APPROVED** [review]
A-OK with me

File: AnalysesFolder/isDecreasingTest/ProcessFlightData.cpp [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+	len = csvLine.find(',');
+	field += csvLine.substr(0, len);
+	csvLine.erase(0, len + 1);
+	return field;
+}
+
+int main() {
+	srand(time(NULL));
+	std::default_random_engine rng(rand());
+	const float STDEV = 1.5f;
```
Comment (bbrk24) [inline AnalysesFolder/isDecreasingTest/ProcessFlightData.cpp:47]: I assume this value is still waiting on @TheRoyalIrishman's analysis?

**Review by bbrk24 — COMMENTED** [review]
LGTM other than my one question

File: AnalysesFolder/isDecreasingTest/ProcessFlightData.cpp (hunk 1 above)
Comment (DarthSethious1) [reply AnalysesFolder/isDecreasingTest/ProcessFlightData.cpp:47]: yup, just left it at 1.5, as 3 standard deviations out is considered to be all points, and our largest was 4.5 ft. Whenever Cam gives us numbers we'll update

Comment (CClarke513) [reply AnalysesFolder/isDecreasingTest/ProcessFlightData.cpp:47]: 1.4432132 is approximately our standard deviation

**Pull request merged**